lib/net/ssh/authentication/ed25519.rb

```diff
@@ -1,4 +1,4 @@
-gem 'rbnacl', '>= 3.2.0', '< 5.0'
+gem 'rbnacl', '>= 3.2.0'
```

Comment:
i'm not sure if we want to allow any future rbnacl, in theory the might not be compatible, on the otherhand they'll be likely compatible for what we use.

Reply:
If it works I don't see how we should freeze it; maybe we can block it `'>= 3.2.0', '< 6.0'`. What do you think?